src/gui/ScreenHandler.hpp

```diff
@@ -53,8 +59,11 @@ class Screens {
     static void RInit(const ScreenFactory::Creator *begin, const ScreenFactory::Creator *end); // init in reversed order, skips nullptr
 
     static Screens *Access();
+    static bool menu_timeout_enabled;
```

Comment:
Make it private

Reply:
also singleton should not have static member variables

Reply:
replace to having Enable/DisableMenuTimeout. It was just shorter to have Screens::menu_timeout = true/false